com.chartboost.mediation/Runtime/Platforms/ChartboostMediationExternal.cs

```diff
@@ -83,12 +84,24 @@ public virtual void SetTestMode(bool testModeEnabled)
         public virtual void DiscardOversizedAds(bool shouldDiscard)
             => Logger.Log(LogTag, $"DiscardOversizedAds : {shouldDiscard}");
 
-        public virtual ChartboostMediationAdapterInfo[] InitializedAdaptersInfo()
+        public virtual ChartboostMediationAdapterInfo[] InitializedAdaptersInfo
         {
-            Logger.Log(LogTag, "InitializedAdaptersInfo");
-            return Array.Empty<ChartboostMediationAdapterInfo>();
+            get
+            {
+                Logger.Log(LogTag, "InitializedAdaptersInfo");
```

Comment:
Should we change this too since the public facing call is changed to `AdaptersInfo` ?
```suggestion
                Logger.Log(LogTag, "AdaptersInfo");
```